enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionDefaultView/InteractiveQuestionDefaultView.tsx

```diff
@@ -71,23 +71,27 @@ export const InteractiveQuestionDefaultView = ({
     withDownloads,
   } = useInteractiveQuestionContext();
 
-  const isCreatingQuestionFromScratch =
-    originalId === "new" && !question?.isSaved();
+  const isNewQuestion = originalId === "new";
+  const isQuestionSaved = question?.isSaved();
```

Comment:
Separate these two, since we need the `useEffect` to be a bit more granular for closing the notebook editor.